docs/cni-plugin.md

```diff
@@ -17,7 +17,9 @@ Please note that Open vSwitch must be installed and running on the host.
     "name": "mynet",
     "type": "ovs",
     "bridge": "mynet0",
-    "vlan": 100
+    "vlan": 100,
+    "mtu": 9000,
+    "trunk": [ { "id" : 42 }, { "minID" : 1000, "maxID" : 1010 } ]
```

Comment:
Is it valid to have both "vlan" and "trunk" set at the same time?

Reply:
Good point; probably not.  I was just putting all the fields in there, which is misleading at best.

I can split it into 2 examples. 👍